src/libraries/Directory.Build.props

```
@@ -118,6 +118,9 @@
     <!-- We build linux-musl-arm on a ubuntu container, so we can't use the toolset build for alpine runtime. We need to use portable linux RID for our toolset in order to be able to use it. -->
     <ToolRuntimeRID Condition="'$(_runtimeOS)' == 'linux-musl' and $(TargetArchitecture.StartsWith('arm')) and !$(HostArch.StartsWith('arm'))">linux-x64</ToolRuntimeRID>
 
+    <!-- cross-targeting FreeBSD on Linux -->
+    <ToolRuntimeRID Condition="'$(_runtimeOS)' == 'freebsd' and $([System.Runtime.InteropServices.RuntimeInformation]::OSDescription.StartsWith('Linux'))">linux-x64</ToolRuntimeRID>
```

```suggestion
    <ToolRuntimeRID Condition="'$(TargetOS)' == 'freebsd' and $([System.Runtime.InteropServices.RuntimeInformation]::OSDescription.StartsWith('Linux'))">linux-x64</ToolRuntimeRID>
```

_runtimeOS, will not be `freebsd`.

---

I tried that but my local build is still failing @safern. I need to find more time to debug this.

---

Ok, ping me offline if you need any help figuring out why.